datajunction-clients/python/datajunction/builder.py

```diff
@@ -460,7 +474,7 @@ def create_metric(  # pylint: disable=too-many-arguments
                 "mode": mode,
             },
             update_if_exists=update_if_exists,
-        )
+        )  # type: ignore
```

Comment:
I don't think the `# type: ignore` is doing anything on this line, so it can be removed

Reply:
that's true